tests/avro/test_avro_serializer.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+#
+# Copyright 2018 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+#
+# derived from https://github.com/verisign/python-confluent-schemaregistry.git
+#
+
+import struct
+
+import unittest
+
+from tests.avro import data_gen
+from confluent_kafka.avro.serializer import AvroSerializer, AvroDeserializer
+from confluent_kafka.avro.schema import GenericAvroRecord
+from tests.avro.mock_schema_registry_client import MockSchemaRegistryClient
+from confluent_kafka import avro
+
+
+class TestAvroSerializer(unittest.TestCase):
+    def setUp(self):
```

Comment:
it would be good also to supply off-the-shelf implementations for other common data types in this PR. amongst other things, it will validate the generality of the approach taken.

Reply:
nit: setUp casing seems atypical to me, setup.

Reply:
ack
ack